addons/tooltips/script.js

```diff
@@ -0,0 +1,613 @@
+let regexDict = {
+  project: [/^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/projects\/(\d+)/],
+  studio: [/^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/studios\/(\d+)/],
+  user: [/^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/users\/([a-zA-Z0-9-_]+)/],
+  forumTopic: [/^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/discuss\/topic\/(\d+)(?!.+#post-\d)/],
+  forumPost: [
+    /^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/discuss\/post\/(\d+)/,
+    /^(?:(?:https?:)?\/\/scratch\.mit\.edu)?\/discuss\/topic\/\d+(?:.+#post-(\d+))/,
+  ],
+};
```

Comment:
i wonder if there's a way to fetch these from get-userscripts.js 🤔

Reply:
That thing is made so it matches both /projects/1234 and https://scratch.mit.edu/projects/1234. I think those on `get-userscripts.js` only support the latter